ios-baseUITests/ios_baseUITests.swift

```diff
@@ -31,28 +33,103 @@ class ios_baseUITests: XCTestCase {
     XCTAssertFalse(signUpButton.isEnabled)
     
     app.type(text: "automation@test", on: "EmailTextField")
-    XCTAssertFalse(signUpButton.isEnabled)
     
-    toolbarDoneButton.tap()
+    toolbarDoneButton.forceTap()
     app.type(text: "holahola",
              on: "PasswordTextField",
              isSecure: true)
     XCTAssertFalse(signUpButton.isEnabled)
     
-    toolbarDoneButton.tap()
+    toolbarDoneButton.forceTap()
     app.type(text: "holahola",
              on: "ConfirmPasswordTextField",
              isSecure: true)
     XCTAssertFalse(signUpButton.isEnabled)
     
-    toolbarDoneButton.tap()
+    toolbarDoneButton.forceTap()
     app.type(text: ".com", on: "EmailTextField")
     XCTAssert(signUpButton.isEnabled)
-    toolbarDoneButton.tap()
+    toolbarDoneButton.forceTap()
     
     app.type(text: "holahol",
              on: "ConfirmPasswordTextField",
              isSecure: true)
     XCTAssertFalse(signUpButton.isEnabled)
   }
+  
+  func testSignInSuccess() {
+    app.launch()
+    
+    app.logOutIfNeeded(in: self)
+    
+    app.attemptSignIn(in: self,
+                      with: "automation@test.com",
+                      password: "holahola")
+    
+    //TEST CODE ONLY ELSE BLOCK SHOULD BE EXECUTED NORMALLY
+    sleep(10)
+    if let alert = app.alerts.allElementsBoundByIndex.first {
+      waitFor(element: alert, timeOut: 10)
+      XCTAssertTrue(alert.label == "Error")
+      
+      alert.buttons.allElementsBoundByIndex.first?.forceTap()
+    } else {
+      let logOutButton = app.buttons["LogoutButton"]
+      waitFor(element: logOutButton, timeOut: 10)
+      
+      logOutButton.forceTap()
+      
+      let goToSignInButton = app.buttons["GoToSignInButton"]
+      waitFor(element: goToSignInButton, timeOut: 10)
+    }
+  }
+  
+  func testSignInFailure() {
+    app.launch()
+    
+    app.logOutIfNeeded(in: self)
+    
+    app.attemptSignIn(in: self,
+                      with: "automation@test.com",
+                      password: "incorrect password")
+    
+    if let alert = app.alerts.allElementsBoundByIndex.first {
+      waitFor(element: alert, timeOut: 2)
```

Comment:
maybe adding a default timeout?

Reply:
I think there should be but I don't recall exactly